src/components/NotificationsDrawer/DrawerPanelContent.tsx

```diff
@@ -1,359 +1,44 @@
-      <NotificationDrawerHeader onClose={onNotificationsDrawerClose} title="Notifications" className="pf-u-align-items-center">
-        {activeFilters.length > 0 && <Badge isRead>{activeFilters.length}</Badge>}
-        <Dropdown
-          toggle={(toggleRef: React.Ref<MenuToggleElement>) => (
-            <MenuToggle
-              ref={toggleRef}
-              onClick={() => setIsFilterDropdownOpen(!isFilterDropdownOpen)}
-              id="notifications-filter-toggle"
-              variant="plain"
-              aria-label="Notifications filter"
-            >
-              <FilterIcon />
-            </MenuToggle>
-          )}
-          isOpen={isFilterDropdownOpen}
-          onOpenChange={setIsFilterDropdownOpen}
-          popperProps={{
-            position: PopoverPosition.right,
-          }}
-          id="notifications-filter-dropdown"
-        >
-          {filterDropdownItems()}
-        </Dropdown>
-        <BulkSelect
-          id="notifications-bulk-select"
-          items={[
-            { title: 'Select none (0)', key: 'select-none', onClick: () => selectAllNotifications(false) },
-            {
-              title: `Select visible (${activeFilters.length > 0 ? filteredNotifications.length : notifications.length})`,
-              key: 'select-visible',
-              onClick: selectVisibleNotifications,
-            },
-            { title: `Select all (${notifications.length})`, key: 'select-all', onClick: () => selectAllNotifications(true) },
-          ]}
-          count={notifications.filter(({ selected }) => selected).length}
-          checked={notifications.length > 0 && notifications.every(({ selected }) => selected)}
-        />
-        <Dropdown
-          toggle={(toggleRef: React.Ref<MenuToggleElement>) => (
-            <MenuToggle
-              ref={toggleRef}
-              onClick={() => setIsDropdownOpen(!isDropdownOpen)}
-              variant="plain"
-              id="notifications-actions-toggle"
-              aria-label="Notifications actions dropdown"
-              isFullWidth
-            >
-              <EllipsisVIcon />
-            </MenuToggle>
-          )}
-          isOpen={isDropdownOpen}
-          onOpenChange={setIsDropdownOpen}
-          popperProps={{
-            position: PopoverPosition.right,
-          }}
-          id="notifications-actions-dropdown"
-        >
-          <DropdownList>{dropdownItems}</DropdownList>
-        </Dropdown>
-      </NotificationDrawerHeader>
-      <NotificationDrawerBody>
-        <NotificationDrawerList>{renderNotifications()}</NotificationDrawerList>
-      </NotificationDrawerBody>
+    <NotificationDrawer ref={panelRef}>
+      <PanelContent></PanelContent>
     </NotificationDrawer>
   );
 };
 
-const DrawerPanel = React.forwardRef((props, innerRef) => <DrawerPanelBase innerRef={innerRef} {...props} />);
+const DrawerPanel = React.forwardRef<unknown, Omit<DrawerPanelProps, 'panelRef'>>((props, innerRef) => (
```

Comment:
After passing down the toggleDrawer func, the typing here is horrible. Let me know if you have a better idea haha

Reply:
I think you can yous HTMLElement for the ref and be done with it. Just to be clear, the notification PR has to be merged first and available probably both in stage production so we are not blocking chrome deployment.